### ISSUE TITLE
Invalid parameters for command "mkdir"

### ISSUE THREAD

**Issue by lasotaartur** [helios-ag/FMElfinderBundle#288]
##### Issue Type:

 - Bug Report

##### Bundle Version:

Master

##### Summary:

When I try upload folder than I get response 
```
Invalid parameters for command "mkdir"
```
json
```
{"error":["errCmdParams","mkdir"]}
```

###### Request
http://127.0.0.1/efconnect?cmd=mkdir&target=l1_Lw&dirs%5B%5D=test&_=1496831023612

###### Query parameters
```
cmd:mkdir
target:l1_Lw
dirs[]:test
_:1496831023612
```

And in https://github.com/helios-ag/ElFinderPHP/blob/master/src/ElFinder.php#L112 parameter `name` is required

**Comment by lasotaartur** [comment]
Somebody can help me ?

**Comment by helios-ag** [comment]
hi @lasotaartur, just checked, want to confirm: this is a bug. Cause is a outdaed ElfinderPHP

**Comment by lasotaartur** [comment]
Yes, this is a bug, but and don't know if this is because of outdated ElfinderPHP.

**Comment by florian25686** [comment]
I have the same problem.

**Installed** 
components/elfinder: 2.2
helios-ag/fm-elfinder-bundle: 7.0.1
helios-ag/fm-elfinder-php-connector: 2.5.7

**Comment by kimwue** [comment]
Same issue, anyone solved this? :)
